systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeAbstractST.java

```diff
@@ -37,40 +27,18 @@
 @Tag(NODEPORT_SUPPORTED)
 @Tag(EXTERNAL_CLIENTS_USED)
```

Comment:
Why do we have these tags if we are removing external access?

Reply:
I forgot to remove it, thanks